src/test/java/teammates/test/cases/browsertests/StudentCourseDetailsPageUiTest.java

```diff
@@ -1,60 +0,0 @@
-package teammates.test.cases.browsertests;
-
-import org.testng.annotations.Test;
-
-import teammates.common.util.AppUrl;
-import teammates.common.util.Const;
-import teammates.e2e.cases.e2e.BaseE2ETestCase;
-import teammates.test.pageobjects.StudentCourseDetailsPage;
-
-/**
- * SUT: {@link Const.ActionURIs#STUDENT_COURSE_DETAILS_PAGE}.
- */
-public class StudentCourseDetailsPageUiTest extends BaseE2ETestCase {
```

Comment:
This should not be removed. Eventually the E2E tests will be migrated as well and the old ones will be the reference.